packages/zoe/src/contracts/coveredCall.js

```diff
@@ -1,84 +1,94 @@
- * escrowing the underlying assets is cancelled. Therefore, the
- * proposal to sell the underlying assets must have an exit record with
- * the key "afterDeadline".
+ * The call option has an expiration date, when the opportunity is
+ * cancelled. The owner of the digital assets cannot remove the assets
+ * from escrow before the expiration date.
  *
- * The invitation received by the covered call creator is the call
- * option. It has this additional information in the invitation's
- * value:
- * { expirationDate, timerAuthority, underlyingAsset, strikePrice }
- *
- * The initial proposal should be:
+ * The `creatorInvitation` of this contract is an invitation to escrow
+ * the underlying assets. The proposal to escrow assets can have any
+ * `give` and `want` with any keywords, but it must have an exit
+ * record with the key "afterDeadline":
  * {
- *   give: { UnderlyingAsset: assetAmount },
- *   want: { StrikePrice: priceAmount  },
- *   exit: { afterDeadline: { deadline: time, timer: timer } },
+ *    give: { ... }, want: { ... }, exit: {afterDeadline: { deadline:
+ *    time, timer: myTimer }
+ *    },
  * }
- * The result of the initial offer is a seat where the payout will
- * eventually resolve either to the strikePrice or to a refund of the
- * underlying asset. The offerResult is the call option, which is an
- * assayable invitation to buy the underlying asset. Since the
- * contract provides assurance that the underlying asset is available
- * on the specified terms, the invitation itself can be traded as a
- * valuable good.
+ *
+ * This deadline serves as the expiration date for the covered call
+ * option. After this deadline, if the option has not been exercised,
+ * the underlying assets are automatically paid out to the creator of
+ * the contract as a refund.
+ *
+ * After the owner of the digital assets escrows the assets in the
+ * initial offer, they receive a seat. The payout for this seat will
+ * either be a refund of the underlying assets (as mentioned above) or
+ * the strike price. Zoe's enforcement of offer safety guarantees that
+ * the payout is either a refund or the strike price, regardless of
+ * whether this contract is buggy.
+ *
+ * The offerResult of this initial seat resolves to the call option
+ * itself: an assayable invitation to buy the underlying assets. The
+ * call option invitation has this additional information in the
+ * value: { expirationDate, timerAuthority, underlyingAssets,
+ * strikePrice }
+ *
+ * The invitation itself can be traded as a valuable digital asset: a
+ * covered call option.
+ *
+ * The recipient of a covered call option (buying it on some other
+ * exchange or through some other trading contract) can exercise the
+ * option before the deadline by using the option as an invitation to
+ * this contract, paying the strike price and receiving the underlying
+ * assets. The recipient of a covered call option can use whatever
```

Comment:
Is it a bonus that they can use whatever keywords? We should figure out whether this is good or bad :).

Reply:
Yup, I think it is clearly a bonus, as this means that the coveredCall can be reused in other contexts, such as being called by another contract without having to remap the keywords.